lib/debug/thread_client.rb

```diff
@@ -337,6 +337,11 @@ def instance_eval_for_cmethod frame_self, src
     end
 
     def frame_eval src, re_raise: false
+      if src == "_ex_"
```

Comment:
The implementation is not acceptable because it can not call methods like `_ex_.cause`.

Reply:
ah you're right. I'll fix it.